gcloud-java-compute/src/main/java/com/google/gcloud/compute/Compute.java

```diff
@@ -541,6 +788,40 @@ public static OperationFilter notEquals(OperationField field, long value) {
   }
 
   /**
+   * Class for filtering address lists.
+   */
+  class AddressFilter extends ListFilter {
+
+    private static final long serialVersionUID = -227481644259653765L;
+
+    AddressFilter(AddressField field, ComparisonOperator operator, Object value) {
+      super(field.selector(), operator, value);
+    }
+
+    /**
+     * Returns an equality filter for the given field and string value. For string fields,
```

Comment:
Nit: equality --> equals (to be consistent with previous javadoc and "not-equals")